backend/lcfs/db/models/compliance/ComplianceReportHistory.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import Column, Integer, ForeignKey
+from sqlalchemy.orm import relationship
+from lcfs.db.base import BaseModel, Auditable
+
+class ComplianceReportHistory(BaseModel, Auditable):
+    __tablename__ = 'compliance_report_history'
+    __table_args__ = (
+        {'comment': "Tracks status changes of compliance reports"}
+    )
+    
+    compliance_report_history_id = Column(Integer, primary_key=True, autoincrement=True, comment="Unique identifier for the compliance report history")
+    compliance_report_id = Column(Integer, ForeignKey('compliance_report.compliance_report_id'), nullable=False, comment="Foreign key to the compliance report")
+    status_id = Column(Integer, ForeignKey('compliance_report_status.compliance_report_status_id'), nullable=False, comment="Foreign key to the compliance report status")
+    user_role_id = Column(Integer, nullable=False, comment="Identifier for the user role associated with the status change")
```

Comment:
Shouldn't this be user_profile_id, similar to transfer history?

Reply:
this was a mistake, should've been user_profile_id.